Orange/widgets/evaluate/owliftcurve.py

```diff
@@ -382,11 +493,30 @@ def set_visual_settings(self, key, value):
         self.visual_settings[key] = value
 
 
-def points_from_results(results, target, clf_index):
-    x, y, thresholds = cumulative_gains_from_results(results, target, clf_index)
-    points = CurveData(x, y, thresholds)
-    hull = CurveData(*convex_hull([(x, y, thresholds)]))
-    return PointsAndHull(points, hull)
+def points_from_results(results, target, clf_index, curve_type):
+    func = precision_recall_from_results \
+        if curve_type == CurveTypes.PrecisionRecall \
+        else cumulative_gains_from_results
+    x, y, thresholds = func(results, target, clf_index)
+    return CurveData(x, y, thresholds)
```

Comment:
How is this better than

```python
if curve_type == CurveTypes.PrecisionRecall:
    x, y, thresholds = precision_recall_from_results(results, target, clf_index)
else:
    x, y, thresholds = cumulative_gains_from_results(results, target, clf_index)
```

:)

If you think it's too long, maybe do `curve = cumulative_gains_from_results(...)` and return `CurveData(*curve)`.